lib/util/copyToClipboard_utils.dart

```diff
@@ -0,0 +1,17 @@
+
+
+import 'package:flutter/services.dart';
+
+class CopyToClipboard{
+
+
+  static Future<void> copyText(String text) async {
+    if (text.isNotEmpty) {
+      Clipboard.setData(ClipboardData(text: text));
+      return;
+    } else {
+      throw ('Please enter a string');
```

Comment:
You can't just throw an exception based on a likely user error like this. The user will see a blank screen and the app will be hosed. Why crash the app?

Reply:
i fixed this on and set onFailure now app is not crashed